test/format/wts.c

```diff
@@ -273,6 +273,14 @@ create_database(const char *home, WT_CONNECTION **connp)
     if (GV(DISK_DATA_EXTEND))
         CONFIG_APPEND(p, ",file_extend=(data=8MB)");
 
+    /* Configure realloc malloc debug mode. */
+    if (GV(DEBUG_REALLOC_MALLOC)) {
+        if (mmrand(NULL, 0, 1) == 1)
+            CONFIG_APPEND(p, ",debug_mode=(realloc_malloc=true,realloc_exact=true)");
+        else
+            CONFIG_APPEND(p, ",debug_mode=(realloc_malloc=true)");
+    }
+
```

Comment:
Does support for REALLOC_EXACT also need to be added?

Reply:
`realloc_exact` is an existing debug_mode setting in the code so I believe this support is already there. Is that what you meant by your comment?

Reply:
@mm-ng, I think @sueloverso's point is for test/format.  You've added a line for `"debug.realloc_exact"` to `test/format/config.sh` .  That's good.  But it means that if someone runs test format, they should be able to individually turn on that option by running: build/test/format/t debug.realloc_exact=1 (or setting it in a CONFIG file).  But that won't actually set the option if there's no code that tests `GV(DEBUG_REALLOC_EXACT)`.

Reply:
Right, got it. Thanks Don!